department-of-veterans-affairs/va.gov-cms
# VAMC-11518 & 11520: VISN 16 and VISN 21 systems menus.

## Description

Map out and Create Admin and Editorial Menus for VISN-16 and VISN-21 Systems

## QA steps

As Admin

- [ ] Visit <Manage/content/menus/VAMC system menus> and validate that menus displayed match this.
<img width="1680" alt="Screen Shot 2020-08-03 at 2 48 13 PM" src="https://user-images.githubusercontent.com/56320995/89218373-eb25ed80-d59b-11ea-90cc-451a6e98b043.png">


Visit the following

- [ ] /admin/structure/types/manage/health_care_local_facility
- [ ] /admin/structure/types/manage/health_care_local_health_service
- [ ] /admin/structure/types/manage/health_care_region_page
- [ ] /admin/structure/types/manage/regional_health_care_service_des
- [ ] /admin/structure/types/manage/vamc_operating_status_and_alerts
and validate that the content types have new Menu items (VISN 16 and 21 systems) selected as enabled (check box) to match this.
<img width="1680" alt="Screen Shot 2020-08-03 at 2 54 47 PM" src="https://user-images.githubusercontent.com/56320995/89218503-258f8a80-d59c-11ea-82ea-45776a7df308.png">

- [ ] Visit "Sections/veterans health administration/VAMC facilities" and validate that menus displayed for VISN 16 and 19 match this.
<img width="1680" alt="Screen Shot 2020-08-03 at 2 56 51 PM" src="https://user-images.githubusercontent.com/56320995/89218555-4526b300-d59c-11ea-88c2-816b413c48ed.png">

- [ ] Visit "Manage/Content/Menus/CMS administration/Administration" and validate that VISN 16 and 21 systems are assigned as child.
<img width="1680" alt="Screen Shot 2020-08-03 at 3 02 49 PM" src="https://user-images.githubusercontent.com/56320995/89218611-6091be00-d59c-11ea-834d-235be8020971.png">

- [ ] Visit "/admin/config/user-interface/menu-breadcrumb" and validate VISN 16 and 21 systems are enabled to match this.
<img width="1680" alt="Screen Shot 2020-08-03 at 3 05 38 PM" src="https://user-images.githubusercontent.com/56320995/89218683-7a330580-d59c-11ea-8ddc-2ad9de946c45.png">


## Definition of Done
